src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -180,21 +179,22 @@ def _watch(self, res_ver):
 
 
 class RookCluster(object):
-    def __init__(self, k8s, rook_env):
+    def __init__(self, coreV1_api, batchV1_api, rook_env):
         self.rook_env = rook_env  # type: RookEnv
-        self.k8s = k8s  # type: CoreV1Api
+        self.coreV1_api = coreV1_api
+        self.batchV1_api = batchV1_api
```

Comment:
```suggestion
        self.coreV1_api: CoreV1Api=coreV1_api
        self.batchV1_api: BatchV1Api=batchV1_api
```
This really helps when using an IDE

Reply:
Sorry... PEP 484 ..... i was going to say something not correct about this "masked beta feature"  but after 50 years i have achieve to have a good self-control.
I only will say. I don't like to do that.

Reply:
we're finally on Python 3 :tada: :tada: :tada: :tada: